librz/util/table.c

```diff
@@ -792,9 +800,15 @@ static int __resolveOperation(const char *op) {
 	if (!strcmp(op, "gt")) {
 		return '>';
 	}
+	if (!strcmp(op, "ge")) {
+		return ')';
```

Comment:
I'm ok with this, but....

why??? why the operation(which is a string) is transformed in a char and then what to do is determined based on those chars? I would understand if these were an enum value...

Reply:
you have to ask the original authors for that, but I think it's simply because it was fast and easy